apps/modernization-ui/src/apps/patient/profile/labReport/LabReportTable.tsx

```diff
@@ -1,17 +1,122 @@
+                    </ClassicLink>
+                ) : null
+            },
+            {
+                id: 2,
+                title: (
+                    <>
+                        {report?.reportingFacility !== null && (
+                            <>
+                                <strong>Reporting facility:</strong>
+                                <br />
+                                <p className="margin0">{report.reportingFacility}</p>
+                            </>
+                        )}
+
+                        {report?.orderingFacility !== null && (
+                            <>
+                                <strong>Ordering facility:</strong>
+                                <br />
+                                <p className="margin0">{report.orderingFacility}</p>
+                            </>
+                        )}
+
+                        {report?.orderingProvider !== null && (
+                            <>
+                                <strong>Ordering provider:</strong>
+                                <br />
+                                <p className="margin0">{report.orderingProvider}</p>
+                            </>
+                        )}
+                    </>
+                )
+            },
+            {
+                id: 3,
+                title: format(report?.collectedOn, 'MM/dd/yyyy') || null
+            },
+            {
+                id: 4,
+                title: report?.results.length ? (
+                    <>
+                        {report.results?.map((result: TestResult) => (
+                            <>
+                                <br />
+                                <strong>{result.test}:</strong>
+                                <br />
+                                <span>{result.result}</span>
+                                <br />
+                            </>
+                        ))}
+                    </>
+                ) : null
+            },
+            {
+                id: 5,
+                title: report?.associatedWith.length ? (
+                    <>
+                        {report.associatedWith?.map((investigation: AssociatedWith, index: number) => (
+                            <div key={index}>
+                                <ClassicLink url={`/nbs/api/profile/${patient}/investigation/${investigation.id}`}>
+                                    {investigation?.local}
+                                </ClassicLink>
+                                <p className="margin-0">{investigation?.condition}</p>
+                            </div>
+                        ))}
+                    </>
+                ) : null
+            },
+            {
+                id: 6,
+                title: report?.programArea || null
+            },
+            {
+                id: 7,
+                title: report?.jurisdiction !== 'null' ? <>{report.jurisdiction}</> : null
```

Comment:
here you can directly pass `report?.jurisdiction !== 'null' ? report.jurisdiction : null`. because it will be handled by the table component